CHANGELOG.md

```diff
@@ -412,7 +414,8 @@ Last release without a changelog :(
 [#1171]: https://github.com/linebender/druid/pull/1171
 [#1172]: https://github.com/linebender/druid/pull/1172
 [#1173]: https://github.com/linebender/druid/pull/1173
-
+[#1186]: https://github.com/linebender/druid/pull/1186
+++
```

Comment:
The text `++` was added here which messes up the formatting of the next section

Reply:
I need a new keyboard :reallyreallysad: I absolutely love this one but the up arrow also maps to + after a couple hours of use.